src/clients/customerCards/get/types.ts

```diff
@@ -0,0 +1,7 @@
+import type { MercadoPagoConfig } from '@src/mercadoPagoConfig';
+
+export declare type CustomerCardsGetRemoveClient = {
```

Comment:
get remove?

Reply:
this interface is shared between the get and remove methods of this client